shared/templates/create_audit_rules_privileged_commands.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python2
+
+#
+# create_audit_rules_privileged_commands.py
+#        generate template-based checks for privileged commands rules
+
+
+from template_common import FilesGenerator, UnknownTargetError
+
+import re
+import ntpath
```

Comment:
do we really need `ntpath` here? Shouldn't this be just `os.path`? See the second note at https://docs.python.org/2/library/os.path.html